mirascope/cli/utils.py

```diff
@@ -227,8 +258,22 @@ def write_prompt_to_template(
     file: str,
     command: Literal[MirascopeCommand.ADD, MirascopeCommand.USE],
     variables: Optional[dict] = None,
-):
-    """Writes the given prompt to the template."""
+) -> str:
+    """Writes the given prompt to the template.
+
+    Deconstructs a prompt with ast and reconstructs it using the jinja2 template, adding
+    revision history into the prompt when the command is `MirascopeCommand.ADD`.
+
+    Args:
+        file: The path to the prompt.
+        command: The CLI command to execute.
+        variables: `None` when the command is `MirascopeCommand.USE` since revision
```

Comment:
The way this is written makes it sound like the function sets the variable based on the command when it's actually that the user of the function provides it.

It would make more sense to explain what variables is, make it clear it's used for add, and if another command is provided it's ignored.